staging/publishing/rules.yaml

```diff
@@ -26,6 +26,11 @@ rules:
       dir: staging/src/k8s.io/code-generator
     name: release-1.13
     go: 1.11.2
+  - source:
+      branch: release-1.14
+      dir: staging/src/k8s.io/code-generator
+    name: release-1.14
+    go: 1.11.5
```

Comment:
we use go 1.12 now https://github.com/kubernetes/kubernetes/pull/74632

sorry for so much back-and-forth, @dims :see_no_evil: